docs/eventing/broker-trigger.md

```diff
@@ -0,0 +1,273 @@
+# Broker and Trigger
```

Comment:
```suggestion
---
title: "Broker and Trigger"
weight: 20
type: "docs"
---
```

Reply:
This is the necessary format for the page to be pulled into the left-nav on the website.

Reply:
Added.